.gitlab/source_test/macos.yml

```diff
@@ -2,39 +2,6 @@
 include:
   - .gitlab/common/macos.yml
 
-tests_macos:
-  stage: source_test
-  # HACK: Run macOS unit tests only on full pipelines, to limit the use of macOS GitHub runners.
-  rules:
-    - !reference [.on_main_or_release_branch_or_all_builds_or_pkg_installer_changes]
-    - when: manual
-      allow_failure: true
-  image: registry.ddbuild.io/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
-  tags: ["arch:amd64"]
-  needs: ["setup_agent_version"]
-  script:
-    - !reference [.setup_macos_github_app]
-    - $S3_CP_CMD $S3_ARTIFACTS_URI/agent-version.cache .
-    - export VERSION_CACHE_CONTENT=$(cat agent-version.cache | base64 -)
-    # Python 3.12 changes default behavior how packages are installed.
-    # In particular, --break-system-packages command line option is
-    # required to use the old behavior or use a virtual env. https://github.com/actions/runner-images/issues/8615
-    - python3 -m pip install -r tasks/libs/requirements-github.txt --break-system-packages
-    - FAST_TESTS_FLAG=""
-    - if [[ "$FAST_TESTS" = "true" ]]; then FAST_TESTS_FLAG="--fast-tests true"; fi
-    - inv -e github.trigger-macos --workflow-type "test" --datadog-agent-ref "$CI_COMMIT_SHA" --version-cache "$VERSION_CACHE_CONTENT" $FAST_TESTS_FLAG --test-washer
-  timeout: 6h
-  after_script:
-    - $CI_PROJECT_DIR/tools/ci/junit_upload.sh "junit-*-repacked.tgz"
-  artifacts:
-    expire_in: 2 weeks
-    when: always
-    paths:
-      - test_output.json
-      - junit-*-repacked.tgz
-    reports:
-      junit: "**/junit-out-*.xml"
-
 .tests_macos_gitlab:
```

Comment:
Should the Gitlab macOS tests run in the MQ instead? With this change, we go from a state where unit tests on macOS run sometimes in the MQ, to a state where they never run in the MQ.

Reply:
Right ! Fixed with [8d5ddc3](https://github.com/DataDog/datadog-agent/pull/32151/commits/8d5ddc31b1cd58ae0a8131f7b744fa2be1f613ab)!